controller_manager:
  ros__parameters:
    update_rate: 10  # Hz
    diagnostic_period: 1

    forward_position_controller:
      type: forward_command_controller/ForwardCommandController

    joint_state_controller:
      type: joint_state_controller/JointStateController

    position_trajectory_controller:
      type: joint_trajectory_controller/JointTrajectoryController

forward_position_controller:
  ros__parameters:
    joints:
      - J0
      - J5
    interface_name: effort

position_trajectory_controller:
  ros__parameters:
    joints:
      - J0
      - J5
    interface_name: position

lss_realtime:
  ros__parameters:
    update_rate: 15  # Hz
    diagnostic_period: 1
